vaadin-context-menu-flow-parent/vaadin-context-menu-testbench/src/main/java/com/vaadin/flow/component/contextmenu/testbench/ContextMenuOverlayElement.java

```diff
@@ -26,27 +26,31 @@
  * <code>&lt;vaadin-context-menu-overlay&gt;</code> element.
  *
  * @author Vaadin Ltd
- *
+ * @deprecated Use {@link ContextMenuElement} instead.
  */
-@Element("vaadin-context-menu-overlay")
+@Element("vaadin-context-menu")
```

Comment:
I wonder if we should just remove `ContextMenuOverlayElement` completely? It was added quite recently.

Reply:
That was my intention as well, but since it was so easy to keep it working I thought why not keep it. We probably won't have to touch it again until we remove it.